third_party/accessibility/gfx/transform.cc

```diff
@@ -1,608 +1,134 @@
-// Copyright (c) 2012 The Chromium Authors. All rights reserved.
```

Comment:
I have to completely rewrite this class because the chromium uses skia to implement this class.